src/repairnator-core/src/main/java/fr/inria/spirals/repairnator/GitRepositoryLauncherUtils.java

```diff
@@ -46,8 +46,7 @@ public static FlaggedOption defineArgGitRepositoryBranch() {
         FlaggedOption opt = new FlaggedOption("gitRepositoryBranch");
         opt.setLongFlag("gitrepobranch");
         opt.setStringParser(JSAP.STRING_PARSER);
-        opt.setDefault("master");
```

Comment:
it's good to have add a default value that is documented. Do we need this change?

Reply:
Hi Martin, sorry for the wait :) . The reason I remove this is due to some branches does not have `master` as Default branch. I saw in `GitCheckoutBranch.java` that if the branch is null (not provided via JSAP) then the GitRepo mode will checkout the default branch automatically.